src/tests/services/reducers/graph-explorer-mode.spec.ts

```diff
@@ -4,7 +4,7 @@ import { Mode } from '../../../types/action';
 
 describe('Graph Explorer Mode Reducer', () => {
   it('should change graph explorer Mode', () => {
-    const initialState = {};
+    const initialState = undefined;
```

Comment:
Why is this set to `undefined`?

Reply:
The `graphExplorerModeReducer` expects the value to be either undefined or of type Mode.TryIt or Mode.Complete.